go.mod

```diff
@@ -6,43 +6,43 @@ require (
 	github.com/Jeffail/gabs v1.4.0
 	github.com/Jeffail/gabs/v2 v2.6.1
 	github.com/ghodss/yaml v1.0.0
-	github.com/imdario/mergo v0.3.12
+	github.com/imdario/mergo v0.3.13
 	github.com/openshift/api v0.0.0-20220908185739-e518ece6c293
 	github.com/openshift/client-go v0.0.0-20220905192401-849f725bff84
-	github.com/openshift/library-go v0.0.0-20220902131052-245d1ca16d15
+	github.com/openshift/library-go v0.0.0-20221021005159-d93563844063
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus-operator/prometheus-operator v0.57.0
-	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.57.0
-	github.com/prometheus-operator/prometheus-operator/pkg/client v0.57.0
-	github.com/prometheus/client_golang v1.12.2
-	github.com/prometheus/prometheus v1.8.2-0.20220315145411-881111fec433 // v1.8.2 is misleading as Prometheus does not have v2 module. This is pointing to v2.34.0, the same as in prometheus-operator v0.57.0
-	golang.org/x/sync v0.0.0-20220722155255-886fb9371eb4
+	github.com/prometheus-operator/prometheus-operator v0.60.1
+	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.60.1
+	github.com/prometheus-operator/prometheus-operator/pkg/client v0.60.1
+	github.com/prometheus/client_golang v1.13.0
+	github.com/prometheus/prometheus v1.99.0 // replaced to v0.39.1 in the section below
+	golang.org/x/sync v0.1.0
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.25.0
+	k8s.io/api v0.25.3
 	k8s.io/apiextensions-apiserver v0.25.0
-	k8s.io/apimachinery v0.25.0
+	k8s.io/apimachinery v0.25.3
 	k8s.io/apiserver v0.25.0
-	k8s.io/client-go v0.25.0
-	k8s.io/klog/v2 v2.70.1
+	k8s.io/client-go v0.25.2
+	k8s.io/klog/v2 v2.80.1
 	k8s.io/kube-aggregator v0.25.0
 	k8s.io/kubectl v0.25.0
 	k8s.io/metrics v0.25.0
-	k8s.io/utils v0.0.0-20220728103510-ee6ede2d64ed
+	k8s.io/utils v0.0.0-20221012122500-cfd413dd9e85
 )
 
 require (
-	github.com/prometheus/common v0.34.0
+	github.com/prometheus/common v0.37.0
 	gopkg.in/yaml.v3 v3.0.1
 )
 
+replace github.com/prometheus/prometheus => github.com/prometheus/prometheus v0.39.1
```

Comment:
Why the `replace` statement? Do we want a reminder here about its purpose?

Reply:
yes, it is to replace the version 1.99 of  github.com/prometheus/prometheus in the require section above.

Reply:
Right, but can you leave a little note here to explain why we need that please?

Reply:
of course 👍

Reply:
updated now